WORKSPACE

```diff
@@ -49,11 +49,10 @@ http_archive(
         "//openxla_patches:cache_urls.diff",
         "//openxla_patches:gpu_race_condition.diff",
         "//openxla_patches:f16_abi_clang.diff",
-        "//openxla_patches:quant_dequant_converter.diff",
     ],
-    strip_prefix = "xla-419a3d736bdfe6891bc40e9ab5c78b466c8e0dc6",
+    strip_prefix = "xla-075d25e0c19e4e455ba0a2bcc432d581128e66aa",
```

Comment:
@JackCaoG @lsy323 I need `libtpu >=20240228`... I will try to move the PIN first before landing this...

Reply:
I will drop pin update in my local branch, waiting for  https://github.com/pytorch/xla/pull/6677 by @lsy323